src/code/bank20.asm

```diff
@@ -4915,9 +4915,18 @@ label_020_6B81:
     ld   [wPaletteUnknownE], a                    ; $6B82: $EA $D5 $DD
     ret                                           ; $6B85: $C9
 
+; Copy two bytes of palette data from bc to hl
```

Comment:
Sounds a lot like thisfunction is part of generating the "fade to white" palette stored in the other wram bank?

Reply:
Indeed it is blending colors.

On further inspection, this is the function used to change the BG palette when a lightning strikes during the intro sequence. I updated the PR with the whole documentation of the thing.